src/Integration/LocalServices/UnboundProjectFinder.cs

```diff
@@ -64,14 +72,20 @@ private IEnumerable<Project> GetUnboundProjects(BindingConfiguration binding)
             var projectSystem = serviceProvider.GetService<IProjectSystemHelper>();
             projectSystem.AssertLocalServiceIsNotNull();
 
-            var filteredSolutionProjects = projectSystem.GetFilteredSolutionProjects();
+            var filteredSolutionProjects = projectSystem.GetFilteredSolutionProjects().ToArray();
```

Comment:
This logging is in a project-type-agnostic class so it will log messages for all bound project types i.e. including C++.